Resources/doc/getting_started/installation.rst

```diff
@@ -0,0 +1,173 @@
+Installation
+============
+
+SonataAdminBundle is just a bundle and as such, you can install it at any
+moment during a project's lifecycle.
+
+1. Download the Bundle
+----------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+.. code-block:: bash
+
+    $ composer require sonata-project/admin-bundle "2.3.*"
+
+This command requires you to have Composer installed globally, as explained in
+the `installation chapter`_ of the Composer documentation.
+
+1.1. Download a Storage Bundle
+------------------------------
+
+You've now downloaded the SonataAdminBundle. While this bundle contains all
+functionality, it needs storage bundles to be able to communicate with a
+database. Before using the SonataAdminBundle, you have to download one of these
+storage bundles. The official storage bundles are:
+
+* `SonataDoctrineORMAdminBundle`_ (integrates the Doctrine ORM);
+* `SonataDoctrineMongoDBAdminBundle`_ (integrates the Doctrine MongoDB ODM);
+* `SonataPropelAdminBundle`_ (integrates Propel);
+* `SonataDoctrinePhpcrAdminBundle`_ (integrates the Doctrine PHPCR ODM).
+
+You can download them in the same way as the SonataAdminBundle. For instance,
+to download the SonataDoctrineORMAdminBundle, execute the following command:
```

Comment:
Why? It's a bundle name, not a code related thing (we are not refering to the class name here specifically).